doc/overview/inspire_overview.rst

```diff
@@ -9,40 +11,33 @@
-environmental policies. This European SDI will enable the sharing of
-environmental spatial information among public sector organisations, facilitate
-public access to spatial information and assist in policy-making across
-boundaries.  INSPIRE is based on the infrastructures for spatial information
-established and operated by the 28 Member States of the European Union. The
-Directive addresses 34 spatial data themes needed for environmental
-applications.  It came into force on 15 May 2007 and will be implemented in
-various stages, with full implementation required by 2021. INSPIRE is on a
-technical level based on international standards, mainly those developed by the
-:doc:`OGC <ogc_overview>` and ISO.
-
-
-Public sector authorities whose data falls within the scope of the Directive shall:
-
-	* Document them through metadata.
-	* Establish network services (discovery, view, download).
-	* Harmonise their spatial data in accordance with common specifications.
-
-Currently, more than 7,000 institutions contribute their data to INSPIRE. A
-significant share of them use OSGeo applications throughout the implementation
-process. :doc:`OSGeoLive <../index>` provides an easy entry to the
-implementation process through an increasing number of readily available
-examples with sample data European data from multiple application domains. In
-addition, a dedicated wiki page on `INSPIRE and OSGeo
-<https://wiki.osgeo.org/wiki/INSPIRE>`__ is established that gives an overview
-of the legal and technical aspects of INSPIRE together with selected use cases
-and an overview of OSGeo tools that implementers may use.
+The `INSPIRE Directive <https://inspire.ec.europa.eu>`__ aims to create a European Union spatial data infrastructure (SDI) for the purposes of environmental policies. This European SDI enables the sharing of environmental spatial information among public sector organisations, facilitates public access to spatial information and assist in policy-making across boundaries. INSPIRE is based on the infrastructures for spatial information established and operated by the 27 Member States of the European Union. The Directive (see the `full text <https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32007L0002&from=EN>`__) came into force on 15 May 2007 and is implemented in various stages, with full implementation required by 2021. INSPIRE is on a technical level based on international standards, mainly those developed by the :doc:`OGC <ogc_overview>` and `ISO <https://www.iso.org>`__.
+
+The datasets falling under the scope of the INSPIRE Directive are grouped into 34 cross-sectoral categories names `spatial data themes <https://inspire.ec.europa.eu/Themes>`__, which are defined in the Annexes of the Directive and reflect two main types of data: geospatial reference data, presented in Annex I and partly Annex II, which define a location reference that the remaining domain-specific themes (in Annex III and partly Annex II) can then refer to.
```

Comment:
although we dont have yet set permanent redirections warnings to be marked as errors we have that plan in the future, as it means that the original page no longer exists. Can you please change
https://inspire.ec.europa.eu/Themes 
to 
https://inspire.ec.europa.eu/Themes/Data-Specifications/2892

Reply:
I dont know why this is shown 2 times